.github/ISSUE_TEMPLATE/new_minor_release.md

```diff
@@ -0,0 +1,67 @@
+---
+name: New Minor Release
+about: Create a issue for tracking a new minor release.
+title: 'Release vX.Y.Z'
+labels: enhancement
+assignees: ''
+
+---
+
+## It is time for a new minor release for Solidus!
+
+We are committed to release every quarter in order to have a constant
+and predictable cadence for our deliverables.
+
+For more information, check out our [Release Policy](https://solidus.io/release_policy/) page.
+
+## Detailed list of steps for the release
+
+We have a pretty clear process to release a new minor version. Everything is described on our
+[Release pipeline](https://github.com/solidusio/solidus/wiki/Release-pipeline-automation) automation and [How to release Solidus](https://github.com/solidusio/solidus/wiki/How-to-release-Solidus) wiki pages,
+which we encourage to review if the assignee is not familiar with how to release Solidus.
+
+### Release preparation
+
+- Be sure there are no pending PRs that you want to include in the next minor.
+- Visit the GitHub Solidus repository.
+- Navigate to the "Actions" tab and select the "Prepare release" item.
+- Click the "Run Workflow" button, leaving `main` as the branch and the checkbox to prepare the next major unchecked (unless necesasy).
+
+At this point a new auto-generated PR will appear; be sure everything looks good and merge it.
+
+### Release the Solidus gems on Rubygems
+
+To officially release the Solidus gems, you need to have permissions on RubyGems, with 2FA enabled.
+
+Open Solidus on your local machine and run:
+
+```bash
+git fetch -a
+git checkout main
+gem release
+```
+
+You will be prompted to enter your 2FA code multiple times, one for each solidus' subgems.
+
+### After-release Chores
+
+Now that Solidus has been released, we need to cleanup the `main` branch to get ready for
+the next release. As you may have noticed, when the "Release Preparation" PR has been merged,
+a new PR has been automatically created. That PR already contains the code to get our repository
+clean and ready. If everything looks green, let's merge it!
+
+### Steps to do outside the main repository
+
+- [ ] [[Guides](https://github.com/solidusio/edgeguides/)] Publish upgrade instruction for the new version ([e.g. PR](https://github.com/solidusio/edgeguides/pull/78)).
+- [ ] [[Starter Frontend](https://github.com/solidusio/solidus_starter_frontend)] Create a new matching branch (e.g. v4.1)
+- [ ] [[Starter Frontend](https://github.com/solidusio/solidus_starter_frontend)] In the new branch (e.g. `v3.3`), change the [default solidus target branch in the CircleCI Config](https://github.com/solidusio/solidus_starter_frontend/blob/4fd99d229cefbe22e7a3916a6dcb6a0cc6cbd41b/.circleci/config.yml#L96) to point to the newly created branch.
```

Comment:
These two steps use different versions as examples (v4.1 and v3.3), even if I have to change the default solidus target branch in the CircleCI Config in the previously created branch.
If so, this might confuse the readers.

Reply:
Thanks for spotting that @rainerdema, fixing!